soak-tests-common/src/main/java/com/hazelcast/jet/tests/common/AbstractSoakTest.java

```diff
@@ -90,9 +105,11 @@ protected final void run(String[] args) throws Exception {
         System.exit(0);
     }
 
-    protected abstract void init() throws Exception;
+    protected abstract void init(JetInstance client) throws Exception;
 
-    protected abstract void test() throws Throwable;
+    protected abstract boolean runOnBothClusters();
```

Comment:
Would not be better to not have it `abstract` but rather as method which returns by default `true` and can be overwritten. We can maybe introduce two subclass of this `AbstractSoakTest`, one for running on only one cluster and one for running on both cluster. Tests can extends one of them. However it is just an idea, it is not needed to implement it. WDYT?